charts/base/templates/job.yml

```diff
@@ -18,7 +18,7 @@ spec:
   template:
     metadata:
       annotations:
-        configmap: "{{ .Values.config | toYaml | sha256sum | trunc 8 }}"
+        configmap: "{{ .Values.config | toYaml | sha256sum | trunc 8 }}"  
```

Comment:
extra whitespace can be removed,
job creation condition `{{- if .Values.job -}}` is always true, so we decided to use ".Values.job.image" to identify if job should be created

Reply:
@mrdntgrn sorry, forgot about the condition. Please check again.